drops/helpers/JsonFileHandler.py

```diff
@@ -9,20 +11,88 @@ class JsonFileHandler:
 
     def __init__(self, file_name : str):
         self.file_name = file_name
+        self.data = dict() 
+        self.endpoints = dict() # EndPoints
+        self.endpoints_map= dict() #MAP
+
+    def create_new_supported_file(self):
```

Comment:
This is beginning to blur the lines between "low level json interaction class and mid level supported ends handler". It is pedantic but I would use the heuristic, "if it cares about the values in the json file it should be in SupportedEndsHandler". 

I would accept dealing with this in a future PR if we cut an issue for it